docs/natives.md

```diff
@@ -59,18 +59,38 @@ else {
 
 The `number` function accepts a string, and parses it as a number.
 
+**Syntax:**
+```melon
+num = number(text: str)
+```
+
 ### `str`
 
 The `str` function accepts a value, and converts it into a string.
 
+**Syntax:**
+```melon
+str = str(value: any)
```

Comment:
This kinda creates an issue with the str function. str is already defined as a function, so reassignment to str variable causes the native function to be overridden, being lost in the current program. This might cause issues if people just copy this code snippet. Maybe we should use another format for denoting return types. Like this

`str(value: any): string`

What do you think?

Btw this is a great addition to docs still. Just wanted to avoid future confusions.

Reply:
Yes! This is a much better way. I wan't happy with the way it was before. Changed everything to be this way.